SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/Data.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AppCenter.Data
+{
+    public partial class Data
+    {
+        /// <summary>
+        /// Change the base URL used to make API calls.
+        /// </summary>
+        /// <param name="apiUrl">API base URL.</param>
+        public static void SetApiUrl(string apiUrl)
+        {
+            PlatformSetApiUrl(apiUrl);
+        }
+
+        /// <summary>
+        /// Check whether the Data service is enabled or not.
+        /// </summary>
+        /// <returns>A task with result being true if enabled, false if disabled.</returns>
+        public static Task<bool> IsEnabledAsync()
+        {
+            return PlatformIsEnabledAsync();
+        }
+
+        /// <summary>
+        /// Enable or disable the Data service.
+        /// </summary>
+        /// <returns>A task to monitor the operation.</returns>
+        public static Task SetEnabledAsync(bool enabled)
+        {
+            return PlatformSetEnabledAsync(enabled);
+        }
+
+        /// <summary>
+        /// Read a document.
+        /// </summary>
+        public static Task<Document<T>> Read<T>(string partition, string documentId)
+        {
+            return PlatformRead<T>(partition, documentId);
+        }
+
+        /// <summary>
+        /// Read a document.
+        /// </summary>
+        public static Task<Document<T>> Read<T>(string partition, string documentId, ReadOptions readOptions)
+        {
+            return PlatformRead<T>(partition, documentId, readOptions);
+        }
+
+        /// <summary>
+        /// List (need optional signature to configure page size).
```

Comment:
Don't put TODO in the documentation, use a separate TODO comment

Reply:
https://github.com/Microsoft/AppCenter-SDK-Android/blob/6c5af99e429c57ca9f318f64a5c5c88ae1fa9dd1/sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Storage.java#L173